File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX (Java)/P03_EnNameOfLastDigit.java
import java.util.Scanner;

public class P03_EnNameOfLastDigit {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        long input = Long.parseLong(in.nextLine());

        DigitToWord(Math.abs(input));
    }

    private static void DigitToWord(long n) {
        long lastDigit = n % 10;
        String word = "";
        String[] digitValue = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};

        for (int i = 0; i <= lastDigit; i++) {

            if (lastDigit == i) {
                word = digitValue[i];
            }
        }
        System.out.println(word);
    }
}

File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX/P03.EnNameOfLastDigit.cs
using System;

namespace P03.EnNameOfLastDigit
{
    class Program
    {
        static void Main(string[] args)
        {
            long input = long.Parse(Console.ReadLine());


            NumberToWord(Math.Abs(input));
        }

        static void NumberToWord(long n)
        {
            long lastDigit = n % 10;
            string word = "";

            string[] digitValue = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };

            for (int i = 0; i <= lastDigit; i++)
            {
                if (lastDigit == i)
                {
                    word = digitValue[i];
                }
            }
            Console.WriteLine(word);
        }
    }
}
